createljspeech.py

```diff
@@ -0,0 +1,54 @@
+# This script generates a folder structure for ljspeech-1.1 processing from mimic-recording-studio database
+# Written as first python script (eg. helloworld) by Thorsten Mueller (MrThorstenM@gmx.net) on november 2019 without any warranty
+
+import sqlite3
+import os
+from datetime import datetime
+from shutil import copyfile
+
+# please set following vars matching your environment
+# ---------------------------------------------------
+
+# Directory of mimic-recording-studio (eg: /home/thorsten/mimic-recording-studio)
+directory_base_mrs = "/home/thorsten/tts/mimic-recording-studio"
+# Output directory for ljspeech structure (eg: /tmp/tts)
+directory_base_ljspeech = "/home/thorsten/tts"
+# See /mimic-recording-studio/backend/audio_files/<id> (eg: 3beeae88-0777-2c8c-5c93-2e844a462e26)
+speaker_id = "4aeeae88-0777-2c8c-5c93-2e844a462e49" 
```

Comment:
can we detect this id from the file structure?

Reply:
I updated the script to query the userid from the sqlite db. But this just works if the db is just used by one user.
SELECT uuid FROM usermodel LIMIT 1;

In case there are more user entries in usermodel table i'd need to ask the user for its username to add this condition in the sql query.